### DISCUSSION TITLE
Writing Private tags into little endian, gets converted to UN

### DISCUSSION THREAD

**Discussion by Tony-Jay** [fo-dicom/fo-dicom#1279] [Q&A] [answered]
Newbe question.

I have a Data set which I read in and then I add a private tag , like this 

`   aDataSet.Add(DicomVR.LT, new DicomTag(group, element, privateCreator), privateTagText); `

When I open in my favourite DICOM editor, it looks great.

If I take the same DICOM data set and convert it into little Endian implicit ( using my favourite DICOM editor) and do the same operation as above , the file which is written out now contains a private TAG with type UN

I have read bits of the DICOM standard like 6.2 and https://dicom.nema.org/medical/dicom/current/output/html/part05.html#sect_A.1

And I really am none the wiser !, I was assuming I would not need to worry about any of these details and FO-DICOM would hide this from me

Is this enough information (I am a bit new to this !) 

Best Regards

Tony

**Comment by mrbean-bremen** [comment, marked as answer]
As described in the documentation you link to, Implicit Little Endian means that the VR field is not written. For private tags this means that no information about the VR is saved, so any private tag can only be shown as UN on loading. It is assumed that anyone who reads a private tag will know the VR of that tag and sets it in the software. This is not specific to fo-dicom.

**Reply by Tony-Jay** [reply]
Thanks alot , reading the standard does not mean I understand the implications of it.
I will close the issue on my side that people can't read my private tag (VR is in my conformance statement)
